packages/foreman/foreman/foreman.spec

```diff
@@ -834,8 +834,15 @@ Group: Development/Libraries
 %description build
 Meta package with support for building RPMs in the Foreman release cycle.
 
+%install build
+install -Dpm0644 foreman.attr %{buildroot}%{_rpmconfigdir}/fileattrs/foreman.attr
```

Comment:
Shouldn't these be listed as `SourceN` and then use the `SOURCEN` macro?

Reply:
picky :P